app/src/main/java/com/example/parkingproyect/mvp/model/Parking.java

```diff
@@ -0,0 +1,26 @@
+package com.example.parkingproyect.mvp.model;
+
+import android.util.Log;
```

Comment:
This import is not being used, please remove it.

Reply:
Okey thanks Isma!